data/biobank-directory/molgenis.csv

```diff
@@ -161,4 +161,16 @@ Persons,,role,Role of the Person,text,,false,,Official role of the person.,,,,,,
 Persons,,biobanks,Biobanks,refback,,false,true,Biobanks the person is connected to.,,Biobanks,contact,,,,,,
 Persons,,collections,Collections,refback,,false,true,Collections the person is connected to.,,Collections,contact,,,,,,
 Persons,,networks,Networks,refback,,false,true,Networks the person is connected to.,,Networks,contact,,,,,,
-Persons,,national_node,National Node,ref,,true,,The person originates from this national node.,,NationalNodes,,${description},,,,,
\ No newline at end of file
+Persons,,national_node,National Node,ref,,true,,The person originates from this national node.,,NationalNodes,,${description},,,,,
+Services,,,,,,,,Biobank Capabilities and Services
+Services,,id,ID,string,1,true,,"Unique ID of the service, prefixed with bbmri-eric:serviceID:"
+Services,,name,Service Name,string,2,true,,Name given to the service locally by the biobank staff.,,,,,,,,,
+Services,,acronym,Acronym,string,,false,,Short symbol simplifying communication.,,,,,,,,,
+Services,,description,Description,string,,false,,Full description of the service. If the description of this particular service exists on the internet - the URL to the relevant page is welcome.,,,,,,,,,
+Services,,device,Device,string,,false,,"If the service is provided with the use of a certain device or instrument - especially when it is novel, rare or sophisticated.",,,,,,,,,
+Services,,deviceSystem,Device System,string,,false,,The system or platform describing the functioning of the used device - if needed.,,,,,,,,,
+Services,,tRL,Technology Readiness Level ,string,,false,,"Technology Readiness Level - concept for self-assessment of the service, used in some European projects (https://en.wikipedia.org/wiki/Technology_readiness_level). Can take values from 1 to 9.",,,,,,,,,

```

Comment:
I would use the list from wikipedia as an ontology. Would make things easier to understand for user.

Reply:
I think we should also add a ref_array to an service type ontology, like in this presentation:
https://docs.google.com/presentation/d/1YehXQMGbTVwH_zJbrxW7eVqj1E5QMvtt/edit#slide=id.p4
(Slide 4)
That would make it possible to search for services that offer a specific type of service (e.g. DNA analysis service)